src/main/java/dev/mayuna/nostalgiadiscord/discord/DiscordBot.java

```diff
@@ -106,8 +107,9 @@ public static void handleChatEvent(AsyncPlayerChatEvent event) {
             return;
         }
 
-        String playerName = event.getPlayer().getName();
-        String message = event.getMessage();
+        String playerName = event.getPlayer().getName().replaceAll("@", "");
+        String message = event.getMessage().replaceAll("@", "");
+
 
         textChannel.sendMessage("<**" + playerName + "**> " + message).queue(success -> {
```

Comment:
Why not just use MarkdownSanitizer.escape 
See : https://ci.dv8tion.net/job/JDA/javadoc/net/dv8tion/jda/api/utils/MarkdownSanitizer.html